### ISSUE TITLE
When 3 of 4 peers are stopped, deploy fails if subsequently start 3 peers together

### ISSUE THREAD

**Issue by scottz64** [hyperledger-archives/fabric#2313]
## Description

Deploy after stopping 3 of 4 peers. Then start the 3 peers again together. All queries fail. 
Note: Instead, if start just two of the peers, with ample delays, then the network works (although there are problems in that scenario too, such as #2309 ).

[fail_CAT_37.tar.gz](https://github.com/hyperledger/fabric/files/382312/fail_CAT_37.tar.gz)
## Describe How to Reproduce

Test case CAT_37_S0S1S2_D_I_R0R1R2_IQ.go, using v0.5 IMAGE=3e0e80a
1. Start a new pbft batch network.
2. Stop VP0 VP1 VP2
3. Send request for new deployment to VP3, and receive a transaction ID (70dfc8e6...)
4. Send Invokes.
5. Restart the 3 peers all at once.
6. Send Invokes.
7. Query. All queries fail. The deploy never worked. Here is my testcase error message:

Error extracted from json: res.Error.Message Query failure
POST /chaincode returned code =-32003 message=Query failure
    data=Error when querying chaincode: Error:Failed to launch chaincode spec(Could not get deployment transaction for 70dfc8e6... - LedgerError - ResourceNotFound: ledger: resource not found)

**Comment by corecode** [comment]
it seems that your test case performs a different sequence.  I can see:
1. start network
2. deploy chaincode

Also peer3 doesn't have automatic null requests enabled, but peer0 does.

**Comment by scottz64** [comment]
Yes, this and pretty much all other testcases start the network and deploy and send invoke(s) to each peer, before proceeding with step 2. 

Thanks: I set NULLREQUEST=1 and batchsize=1 for all the peers and reran, with same results.

Also: The deploy IS processed if I restart only 2 nodes (instead of all 3, with all other things being equal), as in CAT_36_S0S1S2_D_I_R0R1_IQ.go.

**Comment by christo4ferris** [comment]
@scottz64 we've migrated to using [Jira](https://jira.hyperledger.org/secure/RapidBoard.jspa?projectKey=FAB&rapidView=7&view=planning) for issue tracking, please open an issue there and close this one. You'll need a [Linux Foundation ID](https://identity.linuxfoundation.org/) to login. 
Thanks
